asset/scss-about/_nav.scss

```diff
@@ -0,0 +1,89 @@
+// mobile_version
+body {
+  background-color: #f7f7f7;
+  margin: 0%;
+}
+
+nav {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  position: relative;
+}
+
+nav,
+.header_text {
+  padding: 1em;
+}
+
+.openPop {
+  cursor: pointer;
+  display: block;
+  color: #f7f7f7;
+}
+
+.mobile_version a {
+  color: #555;
+}
+
+.closeMenu {
+  display: block;
+  position: absolute;
+  top: 20px;
+  right: 20px;
+  color: #ff4f38;
+
+  a {
+    color: #ff4f38;
+  }
+}
+
+.nav-links {
+  display: none;
+  height: 100vh;
+  flex-direction: column;
+  position: fixed;
+  top: 0%;
+  left: 0%;
+  right: 0%;
+  z-index: 10;
+  justify-content: center;
+  align-items: center;
+  background-color: #272a33;
+  transition: top 2s ease;
+  list-style: none;
```

Comment:
- [ ] As the project requirements dictate. Kindly add the corresponding changes to the mobile menu in order to follow the color/pattern of the top section on the home page, or, to make the background transparent.

>A screenshot from your design
![Screenshot4](https://user-images.githubusercontent.com/47625407/124307722-82a70e00-db68-11eb-8604-5d1adb92dbcb.png)

- [ ] On the same area. Please fix the mobile menu on the about page.

>A screenshot from your design
![Screenshot1](https://user-images.githubusercontent.com/47625407/124308192-31e3e500-db69-11eb-911f-20ab52f3cabc.png)

_**Please refer to the review summary and the given design link for more information.**_